releasenotes/notes/asm-iast-report-scrubbing-ef2aa1d74f866656.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    ASM: The vulnerability report now includes a feature to scrub potentially sensitive information. 
+    This scrubbing process looks for common patterns, and it can be further expanded using environment 
+    variables such as [DD_IAST_REDACTION_NAME_PATTERN and DD_IAST_REDACTION_VALUE_PATTERN](https://ddtrace.readthedocs.io/en/stable/configuration.html#DD_IAST_REDACTION_ENABLED)
```

Comment:
Yup, the built [docs](https://output.circle-artifacts.com/output/job/2bce591c-6cbe-4d7d-b555-d8f4c24b54eb/artifacts/0/tmp/docs/_release_notes_all.html#new-features) show the link isn't showing up nicely, here's a suggestion:
```suggestion
    variables such as ``DD_IAST_REDACTION_NAME_PATTERN`` and ``DD_IAST_REDACTION_VALUE_PATTERN``. See the `docs <https://ddtrace.readthedocs.io/en/stable/configuration.html#DD_IAST_REDACTION_ENABLED>`_ for more information.
```